### PR TITLE
elixir in action 1st attempt on review

### PR DESCRIPTION
**Pull request by gemantzu**
Please be gentle on your comments, this is my first attempt :)

### CODE REVIEW

**Comment by doomspork** [comment]
@pjhampton / @nscyclone / @ybur-yug2 any thoughts on this review?  @gemantzu is going to attempt to review a book/screencast/podcast once a month 😎

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Chapters 5 to 11 are probably the most awesome part of the book (and what I believe made the first edition of it so popular amongst elixir developers), a very thorough explanation about OTP and its counterparts (GenServer, Supervision, ETS Tables, OTP Applications) and the toolkit (Observer, Mix, Phoenix).
+
+In the last two Chapters, we can see how to make our app to run on a cluster, release it using Distillery, and maintain it (Debug, Log etc).
+
+## Conclusion
+
+While this is not a good beginners book nor is it a simple read, it is an excellent resource for illustrating the real platform potential, and struggles, that come with Elixir and BEAM.
+
+I strongly suggest each elixir dev that is not a master in the language to buy a hard copy for their desk. If you are on the beginner level, we will explore the options on our following posts.
```
Comment (ybur-yug2) [inline _posts/2018-05-31-elixir-in-action-review.md]: elixir -> Elixir

**Comment by ybur-yug2** [comment]
I think its a bit short, but is solid. Could elaborate more on how the approach works well for the OTP concepts.

File: _posts/2018-05-31-elixir-in-action-review.md (hunk 1 above)
Comment (nscyclone) [reply _posts/2018-05-31-elixir-in-action-review.md]: This occurs multiple times across the file.

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 2]
```diff
@@ -55,18 +55,12 @@ Chapters 5 to 11 are probably the most awesome part of the book (and what I beli
...
 
-In the last two Chapters, we can see how to make our app to run on a cluster, release it using Distillery, and maintain it (Debug, Log etc).
+In the last two Chapters, we can see how to make our app run on a cluster, release it using Distillery, and maintain it (Debug, Log etc). The author follows the same approach here as well, albeit not in that much detail, as the topics discussed in this section are very broad (clustering, data replication, network partitions, releases) and would need way too much detail to cover in a language introduction book.
 
 ## Conclusion
 
 While this is not a good beginners book nor is it a simple read, it is an excellent resource for illustrating the real platform potential, and struggles, that come with Elixir and BEAM.
```
Comment (pjhampton) [inline _posts/2018-05-31-elixir-in-action-review.md]: You mention here it's not a beginners book, which is fine - but if I was a beginner I wouldn't want to read down multiple paragraphs to get here. Maybe we could have a table at the top summaring the book such as:

```
Level: Beginner | Intermediate | Advanced
Author: XXX
Release Date: XX-20XX
Publisher: XXX
Edition: XXX
...etc
```

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 3]
```diff
@@ -55,18 +55,12 @@ Chapters 5 to 11 are probably the most awesome part of the book (and what I beli
 * add a cache so that we start supporting multiple to-do lists
 * add basic data persistence
 * add a supervision tree and introduce a basic pool of workers
-Additionally, we are introduced to ETS tables, OTP Applications and configuration, dependencies and Phoenix Framework. ** I will add more here **
+Additionally, we are introduced to ETS tables, OTP Applications and configuration, dependencies and Phoenix Framework (as a simple frontend of the application, the author does not go in too much depth on the framework). In this section, I got the feeling that I am pair programming as a junior with the author, sitting beside him while he designs the application. When he finds a hole in his design, he questions himself "What part of the OTP can cover this hole, and how?", stopping each time to turn to me and give me a detailed explanation on what we are doing. He will go on and implement it, and then turn to me again and discuss on alternatives and ups and downs of the chosen path. I like this aproach, and I want to see it in more language learning books.
```
Comment (pjhampton) [inline _posts/2018-05-31-elixir-in-action-review.md]: This is a good review, however the grammar needs polished a little:

```
Additionally, we are introduced to ETS tables, OTP Applications and configuration, dependencies and the Phoenix Framework (as a simple frontend of the application, the author does not go in too much depth on the framework). In this section, I got the feeling that I was pair programming as a junior with the author, sitting beside him while he designs the application. When he finds a hole in his design, he questions himself "What part of the OTP can cover this hole, and how?", stopping each time to turn to me and give me a detailed explanation of what we are doing. He goes on and implement sit, and then turn to me again and discuss alternatives and ups and downs of the chosen path. I like this approach, and I want to see it in more language learning books.
```

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 4]
```diff
@@ -55,18 +55,12 @@ Chapters 5 to 11 are probably the most awesome part of the book (and what I beli
...
 While this is not a good beginners book nor is it a simple read, it is an excellent resource for illustrating the real platform potential, and struggles, that come with Elixir and BEAM.
 
-I strongly suggest each Elixir dev that is not a master in the language to buy a hard copy for their desk. If 
-
-
-
-
-
-are on the beginner level, we will explore the options on our following posts.
+I strongly suggest each Elixir dev that is not a master in the language to buy a hard copy for their desk. If you are on the beginner level, we will explore the options on our following posts.
```
Comment (pjhampton) [inline _posts/2018-05-31-elixir-in-action-review.md]: I'm not sure this adds to the review - it's more of a promotional statement for manning hehe 😄

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 5]
```diff
@@ -40,8 +40,8 @@ It's even harder when that book needs to encompass not only the language but the
...
 
 With this book Saša Jurić has taken another approach: there is a very small project, a ToDoList application, that exists solely to reinforce the readers understanding of each chapter's topics.
-He begins every section explaining the language feature, introducing how and why it is used in the sample project, and lastly finishing the section discussing his experiences and perspective with the particular topics. This is done in a quite natural way, each section is (almost) never broken, making our brain focus at one thing at any given time.
-This allows us, the readers, to experience the ups and downs and explore the content alongside Saša.
+
+He begins every section explaining the language feature, introducing how and why it is used in the sample project, and lastly finishing the section discussing his experiences and perspective with the particular topics. This is done in a quite natural way, each section is (almost) never broken, making our brain focus at one thing at any given time. This allows us to experience the ups and downs and explore the content alongside Saša.
```
Comment (pjhampton) [inline _posts/2018-05-31-elixir-in-action-review.md]: Replace 'He' with 'The Author' throughout

**Comment by gemantzu** [comment]
@pjhampton that's a nice touch, I think I should add something in the conclusion about the personal gains from the book. Does it feel like I am trying to sell the book? That was not my intention, I just enjoyed reading it (while I didn't think I would).

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+ This is our first review of a book, and what a book it is!
+---
+
+## Intro
+
+Welcome to the first post in our new series reviewing various learning material available for Elixir.
+Of course you're probably thinking "Elixir School is the best!" and we agree, but as we explore other material to improve our own we want to share it with you along with our thoughts.
+To get us started, we're going to kick off our series with a review of Saša Jurić's newest book _Elixir in Action, Second Edition._
+
+_Note 2018-18-05: The book is in beta (MEAP - Manning Early Access Program), very close to production according to the Manning website (June 2018).
```
[the remark this answers is not in the document]
Comment (gemantzu) [reply _posts/2018-05-31-elixir-in-action-review.md]: Yeah, although publication date has changed to September.

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+## Chapter Analysis
+
+In total the book has 13 chapters.
+
+In Chapters 1 to 4, we are introduced to the platform, the language and the basics that we can use and find in almost any language (even more on functional ones). We can find details about variables, operators, control flow, data abstractions and protocols (Elixir version of polymorphism). In this section, we begin working on our ToDoList application, and by the end of it, we have a module that contains the CRUD functionality of a ToDoList entity.
+
+Chapters 5 to 11 are probably the most awesome part of the book (and what I believe made the first edition of it so popular amongst Elixir developers), a very thorough explanation about OTP and its counterparts (GenServer, Supervision, ETS Tables, OTP Applications) and the toolkit (Observer, Mix, Phoenix). In this section a lot of features are added to the ToDoList app:
```
Comment (doomspork) [inline _posts/2018-05-31-elixir-in-action-review.md]: There's one last reference to Phoenix here, do we want to remove it @gemantzu?

**Comment by doomspork** [comment]
@ybur-yug2 / @pjhampton any final thoughts?  Waiting to hear back about free copies to give away with the review 👍

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+If you're an experienced developer looking to advance your understanding of Elixir and the underlying BEAM: yes.
+
+This IS NOT a beginner's book.
+This IS NOT a simple book.
+This IS an excellent resource highlighting the potential and shortcomings of Elixir and the BEAM.
+
+If you give it your attention you'll finish the book with a solid understanding of the platform and how best to leverage it at all stages of the development lifecycle.
+
+Look for more reviews to come soon!
```
Comment (doomspork) [inline _posts/2018-05-31-elixir-in-action-review.md]: How do you feel about us redoing the bottom portion here?  Additionally we can edit the post to include the specific twitter URL after we send it out.

```markdown
## Giveaway

To kick off our new series we are giving away free copies of Saša Jurić's _Elixir in Action, Second Edition._!

Want your chance to win a free copy?
Follow us on [Twitter](https://twitter.com/elixirschool) and retweet this blog post announcement!
We'll be picking winners at the end of the month, don't miss your chance!

Look for more reviews to come soon!
```

File: _posts/2018-05-31-elixir-in-action-review.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+author_link: https://github.com/gemantzu
+categories: review
+date:   2018-05-31
+layout: post
+title:  Reviewing Elixir in Action, Second Edition
+excerpt: >
+ The first installment in our new series looking at, and reviewing, different learning materials available to the Elixir community.
+---
+
+## Intro
```
Comment (doomspork) [inline _posts/2018-05-31-elixir-in-action-review.md]: The other posts don't use the `## Intro` header, do we want to remove this?

**Pull request merged**